.github/workflows/ci.yml

```diff
@@ -21,13 +21,13 @@ jobs:
       - uses: actions/checkout@v3
       - uses: goto-bus-stop/setup-zig@v2
         with:
-          version: 0.10.1
+          version: master
```

Comment:
Hello Nitin,

is there a specific reason for targeting `master` instead of `0.11.0`?

I would be more comfortable with continuing to target specific version a upgrading everything in lockstep.

Reply:
Thanks for catching this! `0.11` instead of `master` makes sense.